GoogleForCreators/web-stories-wp
# Fix output overflow regression

## Summary
Fixes regression introduced in #5123 


## Relevant Technical Choices
Adds the background-color directly on the wrapper element of the output -- the same that handles border-radius. This way the background-color and border-radius match as expected.


## To-do



## User-facing changes



## Testing Instructions
- Add a Text element
- Set the background mode as "Fill"
- Add border and border radius.
- Preview the text element -- verify that the background color is not overflowing the borders
- Verify the same for Inner, Center, Outside border


---



Fixes #5324
